src/routes/Landing.js

```diff
@@ -97,18 +99,38 @@ class Landing extends Component {
                             <StackItem className='ins-c-error-state__footer'>
                                 {
-                                    notEntitled.emptyAction.title &&
+                                    notEntitled.emptyAction.primary &&
                                         <Button variant="primary" className='ins-c-error-state__footer-action' onClick={ () => {
-                                            if (notEntitled.emptyAction.navigate) {
-                                                window.location.href = notEntitled.emptyAction.navigate;
+                                            if (notEntitled.emptyAction.primary.navigate) {
+                                                window.location.href = notEntitled.emptyAction.primary.navigate;
                                             }
                                         } } >
-                                            { notEntitled.emptyAction.title }
+                                            { notEntitled.emptyAction.primary.title }
+                                        </Button>
+                                }
+                                {
+                                    notEntitled.emptyAction.secondary ?
+                                        <Level className='ins-c-cmsfrhel-error-state__footer'>
+                                            <LevelItem>
+                                                <Button variant="link" className='ins-c-error-state__footer-close' onClick={ () => {
+                                                    if (notEntitled.emptyAction.secondary.navigate) {
+                                                        window.location.href = notEntitled.emptyAction.secondary.navigate;
+                                                    }
+                                                } }>
+                                                    { notEntitled.emptyAction.secondary.title ?
+                                                        `${notEntitled.emptyAction.secondary.title }` : 'Learn More' }
+                                                </Button>
+                                            </LevelItem>
+                                            <LevelItem>
+                                                <Button variant="link" className='ins-c-error-state__footer-close' onClick={ this.handleModalToggle }>
+                                                    { notEntitled.emptyAction.close ? `${notEntitled.emptyAction.close.title }` : 'Close' }
+                                                </Button>
+                                            </LevelItem>
+                                        </Level> :
+                                        <Button variant="link" className='ins-c-error-state__footer-close' onClick={ this.handleModalToggle }>
+                                            { notEntitled.emptyAction.close ? `${notEntitled.emptyAction.close.title }` : 'Close' }
```

Comment:
This is a whole bunch of code to do something that we don't really need to do. 

Shorten it to be something more like

```
<section className='ins-c-error-state__footer-action--secondary'>
    {
        notEntitled.emptyAction.secondary && notEntitled.emptyAction.secondary.navigate &&
            <Button variant="link" className='ins-c-error-state__footer-secondary' onClick={ () => { window.location.href = notEntitled.emptyAction.secondary.navigate; } }>
                { notEntitled.emptyAction.secondary.title ? `${notEntitled.emptyAction.secondary.title }` : 'Learn More' }
            </Button>
    }
    <Button variant="link" className='ins-c-error-state__footer-close' onClick={ this.handleModalToggle }>
        { notEntitled.emptyAction.close ? `${notEntitled.emptyAction.close.title }` : 'Close' }
    </Button>
</section>
```

* A user shouldn't have a secondary action without some kind of `navigate`, so just check both of those at the same time.
* You don't need a level here since it should be displaying inline anyway, so the buttons should just lay next to each other.